README.md

```diff
@@ -1 +1,33 @@
-# tls-certificates-operator
\ No newline at end of file
+# tls-certificates-operator
```

Comment:
Should we add a not in this `README.md` that this is not recommended for production, since the private key will be in the relation data bucket directly and it might not be secure enough, depending on the environment requirements?

Reply:
Then the question is: "What should we do in production?". We probably want to warn users that private key and certificates are stored in Juju data buckets though.

Reply:
Yes, I agree that a warning would be good enough for now. It lets user decide for themselves what the right security protections are for their situation.